styles/main.css

```diff
@@ -0,0 +1,51 @@
+body {
+  font-family: 'Roboto', serif;
+ }
+
+div.main {
+	margin: 15%;
```

Comment:
Is this a standard value? It seems really high when I'm in full screen but I dunno